kong/plugins/aws-lambda/handler.lua

```diff
@@ -72,14 +123,15 @@ function AWSLambdaHandler:access(conf)
   if err then
     return responses.send_HTTP_INTERNAL_SERVER_ERROR(err)
   end
-
   -- Trigger request
   local client = http.new()
-  client:connect(host, 443)
   client:set_timeout(conf.timeout)
-  local ok, err = client:ssl_handshake()
-  if not ok then
-    return responses.send_HTTP_INTERNAL_SERVER_ERROR(err)
+  client:connect(host, (conf.aws_port or 443))
```

Comment:
We explicitly discussed in #2287 that this form (mixing application and testing code) is undesired. I think we need to find a different approach here (the above PR will solve this)